### ISSUE TITLE
support gist

### ISSUE THREAD

**Issue by leslie-qiwa** [martialblog/cheatsheet#10]
People sometimes also have some gist like multi-line `cheat sheet`. Can you please also consider to add support for it? Thus, this tool can become more generic.

**Comment by martialblog** [comment]
I'll have to look into that. Any Pull Requests are welcome

**Comment by martialblog** [comment]
Ok so I've been thinking about this... It's probably a lot of work to support all kind of Gist stuff. 
Maybe there could be a config for remote-cheatsheets, where you just insert the link to the Raw Remote File and that get's shown on the command line.

**Comment by leslie-qiwa** [comment]
yes, that is what I'm thinking too. people can put any link they want, command line tool just pulls and display for him.

**Comment by martialblog** [comment]
Not going to work on this in the near future
